.github/workflows/build_and_deploy_hold.yml

```diff
@@ -16,16 +16,42 @@ jobs:
     with:
       cloud_region: EU
 
+  merge_multiple_us_artifacts:
```

Comment:
have you tried testing the other build and deploy workflows, particularly build_and_deploy_i18n and build_and_deploy_search_bar? I think they'll all run into the same issue, which is why I suggested having these merge jobs in build_i18n.yml so it can be reused and downstream workflows don't have to worry about it